src/runtime/auto-prompt-manager.js

```diff
@@ -546,7 +550,7 @@ export class AutoPromptManager {
           previouslyShownPrompts.includes(AutoPromptType.CONTRIBUTION_LARGE)
         )
       ) {
-        this.promptDisplayed_ = AutoPromptType.CONTRIBUTION;
+        this.interventionDisplayed_ = {type: AutoPromptType.CONTRIBUTION};
```

Comment:
One thing we should be careful in the future change here.
We're planning to change audience action engine to serve monet prompt type (subscription/contribution) as a part of pre-monet work. It means we'll need to handle audience actions of `AudienceActionType.TYPE_CONTRIBUTION`  in the flow while we used to have a clear distinction between auto-prompts and non-monet action prompts.
We should make it clear that what could be broken with the change. For instance, what if we replace `AutoPromptType.CONTRIBUTION` with `AudienceActionType.TYPE_CONTRIBUTION` here without knowing they are mapped to different string values? Suppression logic can be broken by mistake.

Sorry for long explanation. Can you put a comment to prevent such a situation? (like AutoPromptType.CONTRIBUTION is different from AudienceActionType.TYPE_CONTRIBUTION so should be careful on change)

Reply:
Added a warning regarding difference between `AutoPromptType` and `AudienceActionType`